inference/mlflow-huggingface/src/models/bert-base-uncased/asset.yaml

```diff
@@ -0,0 +1,5 @@
+name: bert-base-uncased
+version: 1
```

Comment:
>1 [](http://example.com/codeflow?start=9&length=1)

auto

Reply:
@adurbhaka should I change this to auto in this iteration?